### DISCUSSION TITLE
Resistors for a noob

### DISCUSSION THREAD

**Discussion by dan88uk** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#90] [Q&A]
Hey everyone. Noob question - what kind of 10k resistor do I need? Looking online there are loads!
Also, some of the pictures people have uploaded of their pcbs have a lot of other resistors mounted, I've used the pcb design in the build guide pdf, does this only need a 10k resistor? Thanks!

**Comment by visualapproach** [comment]
Hi
With the exceptions mentioned in the build instructions, yes. And it's not necessarily needed. As long as it's 10k and fits it doesn't matter what kind you use.

**Reply by dan88uk** [reply]
Awesome! Thanks so much!
